web/src/routes/stores/[...slug].svelte

```diff
@@ -0,0 +1,25 @@
+<svelte:head>
+    <title>Stores</title>
+</svelte:head>
+
+<script context="module">
+    export async function load({ params }) {
+        const { slug } = params;
+        return { props: { storeName: slug } }
+    }
+</script>
+
+<script lang="ts">
+    import { Auction, fromJson } from "$lib/types/auction";
+    import StoreView from "$lib/components/StoreView.svelte";
+    export let storeName: string;
+    let auctions: Auction[] | null = [];
+
+</script>
+
+
+<StoreView
+    loader={{endpoint: `users/${storeName}/auctions`, fromJson}}
```

Comment:
A "store view" would always load a user's auctions, so I don't see why it would receive a "loader", when it could receive a nym and hide the logic. The caller of a StoreView should not need to be aware of the way that view loads whatever it needs.

I made it in the case of a ListView, since a ListView could show anything (auctions, campaigns...) - but a StoreView is something concrete that will not be reused for other things.